sound/soc/soc-topology.c

```diff
@@ -891,7 +891,7 @@ static int soc_tplg_denum_create(struct soc_tplg *tplg, size_t size)
 	se->shift_l = tplg_chan_get_shift(tplg, ec->channel,
 		SNDRV_CHMAP_FL);
 	se->shift_r = tplg_chan_get_shift(tplg, ec->channel,
-		SNDRV_CHMAP_FL);
+		SNDRV_CHMAP_FR);
```

Comment:
should there be a fixes tag for this?

Reply:
Maybe. It was introduced in 8a9782346dcc, however 0db627c4f5df70 removed one indentation level later while simplifying code, so not sure if it will apply cleanly if I do Fixes on commit which introduced it.

Reply:
ok. It looks like a pretty obvious mistake but at the same time if no one complained about it in 10 years then maybe it's not really a problem...

Reply:
You could add a note below the --- mark and let maintainers decide what's best.